src/main/rx/cc2500_frsky_x.c

```diff
@@ -316,6 +318,8 @@ bool isValidPacket(const uint8_t *packet)
     case RX_SPI_FRSKY_X_LBT_V2:
         useBindTxId2 = true;
         break;
+    default:
+        break;
```

Comment:
This will leave `useBindTxId2` undefined. If there are no other cases possible then those defined then an if/else probably makes more sense.

Reply:
More ways to go: but as it seems to be a flag for detecting V2 initializing `useBindTxId2` at declaration we could skip the V1 comparison also and make it a oneliner:
`bool useBindTxId2 = (spiProtocol == RX_SPI_FRSKY_X_V2 || spiProtocol == RX_SPI_FRSKY_X_LBT_V2);`